components/lsp/MintForm.tsx

```diff
@@ -53,11 +58,21 @@ const MintForm: FC<Props> = ({
   shortTokenBalance,
   refetchLongTokenBalance,
   refetchShortTokenBalance,
+  collateralSymbol,
+  showWallet,
+  setShowWallet,
 }) => {
-  const [collateral, setCollateral] = useState("uma");
+  const [collateral, setCollateral] = useState(collateralSymbol);
```

Comment:
I think `collateral` can just be a static prop. The collateral address for an LSP will never change during the component lifecycle, so it can be a static prop

Reply:
Agreed.